splitter/processor.go

```diff
@@ -86,9 +97,41 @@ func (p *Processor) Process(r io.Reader, event *event.FileUploaded, startTime ti
 	})
 }
 
+func sendDatasetSplitEvent(datasetID string, totalRows int) {
+
+	message := DatasetSplitEvent{
+		DatasetID: datasetID,
+		TotalRows: totalRows,
+		SplitTime: time.Now().UTC().Unix() * 1000, // unix time in milliseconds
+	}
+
+	messageJSON, err := json.Marshal(message)
+	if err != nil {
+		log.Error(err, log.Data{
+			"details": "Could not create the json representation of message",
+			"message": messageJSON,
+		})
+		panic(err)
```

Comment:
We should probably not panic here, but it does raise the question what we should do to handle errors. If the file is split but it fails to send the message it will never be complete.